index.html

```diff
@@ -65,6 +65,7 @@
     <?php endif; ?>
     <input type="submit" name="submit" value="Submit">
   </form>
-  </body>
 
   </html>
+
+ </body>
```

Comment:
Not sure the body tag should be moved outside the html tag, can you double check?